pkg/eosclient/eosgrpc/eosgrpc.go

```diff
@@ -47,7 +47,9 @@ import (
 )
 
 const (
-	versionPrefix = ".sys.v#."
+	versionPrefix  = ".sys.v#."
+	userACLEvalKey = "eval.useracl"
```

Comment:
Looks good, but why do you need this key? We shouldn't use it any longer (and I have a PR to remove it from eosbinary as well)

Reply:
Well, I had copied that part from the current master of eosbinary

Reply:
Ah ok! then you can remove it ;)